src/subs.jl

```diff
@@ -55,7 +55,8 @@ function walk_expression(ex)
         x = eval(parse(replace(toString(ex), "I", "im")))
         return Expr(:call, :complex, real(x), imag(x))
     elseif fn == :Constant
-        return constant_map(toString(ex))
+        return constant_map[toString(ex)]
```

Comment:
Thanks for the reminder. Just added a simple one.

On Fri, Apr 8, 2016 at 12:59 PM, Isuru Fernando <notifications@github.com>
wrote:

> In src/subs.jl
> <https://github.com/symengine/SymEngine.jl/pull/21#discussion_r59056362>:
>
> > @@ -55,7 +55,8 @@ function walk_expression(ex)
> >          x = eval(parse(replace(toString(ex), "I", "im")))
> >          return Expr(:call, :complex, real(x), imag(x))
> >      elseif fn == :Constant
> > -        return constant_map(toString(ex))
> > +        return constant_map[toString(ex)]
>
> Is this tested?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/SymEngine.jl/pull/21/files/dcb098671a4db145573d277745ba8dd1515369e6#r59056362>
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu